docs/data/project-metrics-template.md

```diff
@@ -0,0 +1,61 @@
+# Project Metrics
+
+## Introduction
+
+A high-level summary of the project goals.
+
+### Example
+
+This document outlines project objectives, including those related to people, delivery and product. It aims to provide a systematic and quantifiable set of key results that could guide us build better teams, deliver more efficiently and delight our users.
+
+## Objective 1
+
+List the key objective that your project should focus on this Trimester.
+
+**Examples**
+
+- Onboard all new team members successfully
+- Deliver an exceptional training program
+- Improve our team engagement and satisfaction score
+- Launch the new product features successfully
+
+### Key Results
+
+Define the key results (KR) that support the above objective. Please make sure these KRs are specific, measurable and time-bound. They are typically include hard numbers.
+
+**Examples**
+
+If the objective is to successfully launch a new product features by the end of Trimester 1, then the key results are:
```

Comment:
```suggestion
If the objective is to successfully launch a new product feature by the end of Trimester 1, then the key results are:
```

Reply:
(Or "launch new product features" - just need to make the language/article use match :) )